containerd/debug-logging/README.md

```diff
@@ -0,0 +1,13 @@
+# Containerd Debugging Logging
+
+The `containerd-debug-logging-daemonset.yaml` is a daemonset that enables
+containerd debug logs. These logs may be useful for troubleshooting. Note
+that debug logs are quite verbose and such increase log volume for these logs.
+
+To filter which nodes run the daemonset and enable debug log, modify the daemonset to
```

Comment:
The daemonset has a nodeSelector (`containerd-debug-logging: "true"`) already; let's document how to use that.

Reply:
Good point, updated the README, ptal.